content/nginxaas-google/getting-started/create-deployment/deploy-console.md

```diff
@@ -85,22 +85,192 @@ In the NGINXaaS Console,
       - For **Port number**, enter the same port as your NEG's Producer port, for example, port `80`.
 
+
+Each listening port configured on NGINX requires its own network endpoint group with a matching port. You can use the following helper script to automate these steps:
+
+{{< details summary="Show helper script" >}}
+
+   ```bash
+   #!/bin/bash
+   set -euo pipefail
+
+   # Default values
+   PROJECT=""
+   REGION=""
+   NETWORK=""
+   SA_URI=""
+   PORTS="80"
+   PROXY_SUBNET="psc-proxy-subnet"
+   VIPNAME="psc-vip"
+   # Prerequisites:
+   # - gcloud CLI installed and configured
+   # - An existing projectID and a VPC network created in that project
+   # - A valid Service Attachment URI from F5 NGINXaaS
+
+   # Function to display usage
+   usage() {
+      cat << EOF
+      Usage: $0 --project PROJECT --region REGION --network NETWORK --service-attachment SA_URI [--ports PORTS]
+
+      Options:
+         --project                 GCP Project ID
+         --region                  GCP Region
+         --network                 VPC Network name
+         --service-attachment      Service Attachment Self Link
+         --ports                   Comma-separated list of ports (default: 80)
+         --help                    Show this help message
+
+      Note: Proxy subnet and public IP will be automatically created as 'psc-proxy-subnet' and 'psc-vip' respectively.
+
+      Example:
+         $0 --project my-project --region us-central1 --network my-vpc \\
+            --service-attachment "projects/producer-proj/regions/us-central1/serviceAttachments/sa-aa4c6965-4b03-4518-85ea-2ca6fc2e869c" \\
+         --ports "80,443,8080"
+      EOF
+   }
+
+   # Parse command line arguments
+   while [[ $# -gt 0 ]]; do
+      case $1 in
+         --project)
+            PROJECT="$2"
+            shift 2
+            ;;
+         --region)
+            REGION="$2"
+            shift 2
+            ;;
+         --network)
+            NETWORK="$2"
+            shift 2
+            ;;
+         --service-attachment)
+            SA_URI="$2"
+            shift 2
+            ;;
+         --ports)
+            PORTS="$2"
+            shift 2
+            ;;
+         --help|-h)
+            usage
+            exit 0
+            ;;
+        *)
+            echo "Unknown option: $1"
+            usage
+            exit 1
+            ;;
+      esac
+   done
+
+   # Set auto-generated proxy subnet name and VIP name
+
+   # Validate required parameters
+   missing_params=()
+   [[ -z "$PROJECT" ]] && missing_params+=("--project")
+   [[ -z "$REGION" ]] && missing_params+=("--region")
+   [[ -z "$NETWORK" ]] && missing_params+=("--network")
+   [[ -z "$SA_URI" ]] && missing_params+=("--service-attachment")
+
+   if [[ ${#missing_params[@]} -gt 0 ]]; then
+      echo "Error: Missing required parameters: ${missing_params[*]}"
+      usage
+      exit 1
+   fi
+
+   # Create proxy-only subnet (skip if exists)
```

Comment:
this statement can be echoed out instead of being a comment. The comment applies throughout the script where you do so.